package.json

```diff
@@ -36,6 +37,7 @@
   },
   "devDependencies": {
     "chai": "^3.2.0",
+    "compression": "^1.6.0",
```

Comment:
see https://github.com/strongloop/generator-loopback/blob/17e01ccfcf1abd8f37361ee985412ce05e5475c8/test/end-to-end/example.e2e.js